lib/email_sender.py

```diff
@@ -2,6 +2,9 @@
 import datetime
 import smtplib
 from email.mime.text import MIMEText
+import lib.oauth2 as oauth2
+from googleapiclient.discovery import build
+import base64
 
```

Comment:
unrelated, we should update the config.yml.template file

Reply:
config template updated